monshan/overlook-solo
# Architecture/setup

# PR Overlook Solo Project
**Marika Shanahan**

### Fix or Feature?


### Summary of the changes

*Addition of 3 classes with accompanied testing files filled with dummy data and robust happy-sad path testing. This will serve as the base for the javascript logic of the project*

### What ticket(s) do the changes resolve?

- Setup logic architecture
- User class (represents a single user)
- Rooms class (represents all available rooms)
- Bookings class (represents all available bookings)

### Notes for the Reviewer

N/A

### How to Test Changes?

*Run npm test at the head of the directory, should see 20+ passing tests and no failing tests*
